cluster/Vagrantfile

```diff
@@ -1,116 +1,137 @@
-          # make all nics 'virtio' to take benefit of builtin vlan tag
-          # support, which otherwise needs to be enabled in Intel drivers,
-          # which are used by default by virtualbox
-          v.customize ['modifyvm', :id, '--nictype1', 'virtio']
-          v.customize ['modifyvm', :id, '--nictype2', 'virtio']
-          v.customize ['modifyvm', :id, '--nictype3', 'virtio']
-          v.customize ['modifyvm', :id, '--nicpromisc2', 'allow-all']
-          v.customize ['modifyvm', :id, '--nicpromisc3', 'allow-all']
-          v.customize ['modifyvm', :id, '--paravirtprovider', 'kvm']
-        end # v
+      # configure ip address etc
+      c.vm.hostname = vm_name
+      c.vm.network :private_network, ip: node_addr
+      c.vm.network :private_network, ip: node_addr, virtualbox__intnet: 'true', auto_config: false
+      if n.zero?
+        c.vm.network 'forwarded_port', guest: 10_000, host: 10_005 + node_addr.split('.')[3].to_i
+      end
+      c.vm.provider 'virtualbox' do |v|
+        # kube api server node is super slow with just 1GB RAM, so give it more
+        v.memory = ENV['CONTIV_KUBEADM'] && n.zero? ? 2048 : 1024
+        # make all nics 'virtio' to take benefit of builtin vlan tag
+        # support, which otherwise needs to be enabled in Intel drivers,
+        # which are used by default by virtualbox
+        v.customize ['modifyvm', :id, '--nictype1', 'virtio']
+        v.customize ['modifyvm', :id, '--nictype2', 'virtio']
+        v.customize ['modifyvm', :id, '--nictype3', 'virtio']
+        v.customize ['modifyvm', :id, '--nicpromisc2', 'allow-all']
+        v.customize ['modifyvm', :id, '--nicpromisc3', 'allow-all']
+        v.customize ['modifyvm', :id, '--paravirtprovider', 'kvm']
+      end # v
 
+      if ENV['CONTIV_KUBEADM']
         c.vm.provision 'shell', inline: <<-EOS
-          #copy the etc_hosts file we created
-          sudo cp /shared/.etc_hosts /etc/hosts
-          EOS
-        if ENV['VAGRANT_USE_KUBEADM']
-          c.vm.provision 'shell', inline: <<-EOS
             sudo setenforce 0
             sudo systemctl stop firewalld
             sudo /etc/init.d/network restart
-            #copy the etc_hosts file we created
-            sudo cp /shared/.etc_hosts /etc/hosts
             EOS
-          c.vm.provision :shell, path: 'bootstrap_centos.sh'
-          if role == 'master'
-            # Install kubernetes on master
-            ks8_ver = ENV['CONTIV_K8s_VERSION'] || 'v1.4.7'
-            c.vm.provision :shell, path: 'k8smaster_centos.sh', args: [token, member_info['contiv_control_ip'], ks8_ver]
-          else
-            # Install kubernetes on nodes
-            c.vm.provision :shell, path: 'k8sworker_centos.sh', args: [token, master_ip]
-          end # if
+        c.vm.provision :shell, path: 'bootstrap_centos.sh'
+        if n.zero?
+          # Install kubernetes on master
+          ks8_ver = ENV['CONTIV_K8s_VERSION'] || 'v1.4.7'
```

Comment:
One other thing I just noticed... the "s" in the envvar is lowercase.  Is that intentional?